Finished portfolio mobile

Boku52500 opened the pull request:
- Copied first page into a new branch.
- Modified index.html to add new pages.
- Modified style.css to design new pages.
- Added all the pages from figma.
- Fixed all linters.

Boku52500 committed:
Finished portfolio (day 4)

Boku52500 committed:
Fixing linters

Boku52500 committed:
Fixing linters

Boku52500 committed:
Fixing linters

Boku52500 committed:
Update style.css

Boku52500 committed:
Fixing linters

franclobo reviewed (commented):
Hi @Boku52500, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a Pull Request 📑 👀  with a meaningful title and description of the activities in the branch. Kindly replace each section on the README template 🧐 with your information.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

awais-amjed reviewed (commented):
Hi @Boku52500 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted a project, that doesn't have a PR description, kindly add description to your Pull Request 🤗

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

ha-manel commented on index.html:
```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+
+    <section class="2-page">
+        <div class="snapshot">
+            <a href=""><img src="Pictures/snapshot.png" alt=""></a>
```
- [x] Kindly remove the anchor tag `a` around the project screenshot because it's just an image not a link 👍

ha-manel commented on index.html:
```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
```
- [x] Kindly update this `h4` tag to a `p` tag because this is a paragraph to describe the project, and not a header 💯

ha-manel commented on index.html:
```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
```
- [x] Kindly make this section as a list `ul` where each item `li` is one of the technologies 👍

ha-manel commented on index.html:
```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+    <section class="aboutme">
+        <div class="headline">
+            <div class="headline-heading">
+                <h1>About myself</h1>
```
[OPTIONAL]

- [ ] It's better if the webpage has only one `h1` tag while the rest of the subheaders are `h2`, `h3`...

ha-manel commented on index.html:
```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+
+
+    <section class="3-page">
+        <div class="snapshot2">
+            <a href=""><img src="Pictures/snapshot2.png" alt=""></a>
+            <h6>Multi-Post
+                Stories</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
```
- [ ] Kindly try to follow the Figma design as much as possible, this includes card borders, the alignment of the cards (yours are to the left while in Figma they're centered), the buttons (background-color and interactions), the technologies badges also look very different from the ones on Figma.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192100498-206bf70f-f1bd-43ea-927b-a4c47e0637ca.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192100530-83abf547-d45b-4c99-887d-07f4af9f41c2.png)

ha-manel commented on index.html:
```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+                </ul>
+            </div>
+        </div>
+        <button class="but1">Get my resume</button>
```
- [ ] Kindly update the styles on this button to make it match the one on Figma. this includes the background color and the interactions with this button and all the other buttons (hover, pressed...)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192100642-c529075c-be01-4cda-9182-1b1832d9f0d7.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192100653-72e2a652-ae3f-4a5e-8787-b4ef874f8f25.png)
![image](https://user-images.githubusercontent.com/50721479/192100657-6b884233-e5bc-4845-9678-65a64ee05312.png)

ha-manel reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️

## Hi @Boku52500 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  

<img src="https://media0.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif"/>

## Highlights:  
* All tests passed ✔️
* Descriptive PR description ✔️ 

## Required Changes:
- [x] Good job on using the README template provided by Microverse 👏 Kindly edit the empty sections and the placeholder texts and replace them with relevant text and descriptions to your project 💯 

_Kindly check the comments below_

## Optional:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Boku52500 committed:
Update for fixing code.

Boku52500 committed:
Fixing linters

Boku52500 commented:
- Removed unnecessary anchor tags.
- Updated the h4 tag.
- Modified ul li lists.
- Aligned all the media in center.
- Updated the styles on buttons.
- Fixed linters.

christianbiring1 commented on index.html:
```diff
@@ -35,23 +39,232 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+    <section class="aboutme">
+        <div class="headline">
+            <div class="headline-heading">
+                <h1>About myself</h1>
```
[OPTIONAL]
 - [ ] Only one `h1` should suppose to be on the page, Kindly make sure you take that into consideration.

christianbiring1 reviewed (approved):
Hi @Boku52500 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
![APPROVED](https://user-images.githubusercontent.com/97100091/192114928-a4f80386-d020-4e55-aa5b-827f65d4f76d.gif)

## Highlights
- [ ] All linters are passing
- [ ] the project is documented in a professional way
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
